DEVELOPMENT.howto.md

```diff
@@ -43,7 +43,12 @@ We expect Shaderc to always build and test successfully on the platforms listed
 below.  Please keep that in mind when offering contributions.  This list will
 likely grow over time.
 
-* Linux x86
+| Platform | Build Status |
+|:--------:|:------------:|
+| Android (ARMv7)  | |
```

Comment:
Should we leave the Android status mysteriously blank, say something like "not automated," or list the status of the most recent manual build?